apps/ledger-live-desktop/tests/specs/speculos/receive.address.spec.ts

```diff
@@ -49,3 +61,31 @@ for (const account of accounts) {
     );
   });
 }
+
+test.describe("Receive", () => {
+  const account = Account.TRX_2;
+  test.use({
+    userdata: "speculos-tests-app",
+    speculosApp: account.currency.speculosApp,
+  });
+  test(
+    `${account.currency.ticker} empty balance Receive displays address activation warning message`,
+    {
+      annotation: {
+        type: "TMS",
+        description: "B2CQA-1551",
+      },
+    },
+    async ({ app }) => {
+      await addTmsLink(getDescription(test.info().annotations).split(", "));
+
+      await app.layout.goToAccounts();
+      await app.accounts.navigateToAccountByName(account.accountName);
+      await app.account.expectAccountVisibility(account.accountName);
+
+      await app.account.clickReceive();
+      await app.modal.continue();
```

Comment:
Could we factorise until this part as it is the same steps as in previous test

Reply:
created [this ticket](https://ledgerhq.atlassian.net/browse/QAA-196) in order for us to think about a "step orchestrator" to access any step from any test suite